src/aleph_vrf/coordinator/vrf.py

```diff
@@ -336,3 +339,67 @@ async def get_existing_vrf_message(
         return None
 
     return messages[0]
+
+
+async def get_existing_message(
+    aleph_client: AuthenticatedAlephClient,
+    item_hash: ItemHash,
+) -> Optional[PostMessage]:
+    logger.debug(
+        f"Getting VRF message on {aleph_client.api_server} for item_hash {item_hash}"
+    )
+
+    message, status = await aleph_client.get_message(
```

Comment:
What happens if the status is not OK ? Should that case be handled here ?

Reply:
I just added the status control to ensure that is processed.

Reply:
Finally I had to remove the control, because the SDK already makes the control, so the get_message method, don't return any status. If the status is forgotten, it raise an exception.